integrations/overview.mdx

```diff
@@ -0,0 +1,30 @@
+Benji Integrations allow you to connect your payment processor to Benji, enabling seamless loyalty and rewards experiences for your customers.
```

Comment:
Benji Integrations allow you to connect your payment processor --> Benji Integrations allow you to connect your payment processor, e-commerce framework or loyalty platform....